### ISSUE TITLE
[accessibility] Controls should be accessible

### ISSUE THREAD

**Issue by Malvoz** [StephanWagner/svgMap#28]
- Controls should be keyboard accessible:
  - Focusable
  - Styles to convey that a control has focus (e.g. through CSS `outline`)
  - Activate on both <kbd>Enter</kbd> and <kbd>Space</kbd> keys per W3C [ARIA Authoring Practices recommendation](https://www.w3.org/TR/wai-aria-practices-1.1/#keyboard-interaction-3)
- Controls should have [accessible names](https://web.dev/aria-name/) such that ATs (e.g. screen readers) can convey the purpose of the control to the user
